install/config.json

```diff
@@ -0,0 +1,52 @@
+{
+    "requirements": "install/requirements.txt",
+    "requirements-hw": "install/requirements-hw.txt",
+    "aptget": [
+        "gcc",
+        "python",
+        "python-dev",
+        "python-pip"
+    ],
+    "aptget-hw": [
+        "python-smbus",
+        "libffi-dev",
+        "i2c-tools",
+        "wiringpi",
+        "python-numpy"
+    ],
+    "aptget-cam": [
+        "libjpeg8-dev",
+        "imagemagick",
+        "libv4l-dev",
+        "cmake"
+    ],
+    "virtualenv": "~/.local/bin/virtualenv",
```

Comment:
This looks to be the path to the virtualenv binary, but I'm not sure this is true for everyone. I think I have it installed globally. As long as `virtualenv` is in your `$PATH` before running scotty it should work. Did you have an issue where it wasn't?

Reply:
So the issue is that we run scotty under SUDO and I guess the $PATH for sudo doesn't contain virtualenv's path, so it has to be stated on the call to it.

Reply:
Hmm, okay. I guess just be wary of this, I'm giving it a higher than 50% chance that the path will be wrong for someone else.

Reply:
Probably, but I'll let them make that modification in their config.json. The only other way is to dynamically get the path by making another call with `whereis virtualenv`?

Reply:
I believe `whereis` just looks in your path, so if it's not there it won't help. However, maybe you could use it to test if `virtualenv` is in the path and if not use the config.json path. That may give it the best chance of working (I don't remember it being a problem last year, so in some configuration it works without the hard coded path) 

If people change their config.json, then it will probably get merged along with their PR's whenever they change code.

Reply:
Okay, that sounds like the best option right now.

Reply:
We can add it to the .gitignore file now, I believe, and it should download a default to their systems while ignoring any changes they make to the config file.

And we can put instructions in the README for anyone that just wants to ignore that and just ask us anyways

Reply:
Right. I still need to put stuff in the README...

Reply:
Consider yourself granted a grace period.... just know I'm not letting you live this down til you do 😈